text-generation-inference/tests/test_prefill_truncate.py

```diff
@@ -1,35 +1,47 @@
+import pytest
 from helpers import create_request, prepare_model
 from text_generation_server.auto_generator import AutoGenerator
 from text_generation_server.pb.generate_pb2 import Batch
 
+from optimum.tpu.jetstream_pt_support import jetstream_pt_available
 
-def test_prefill_truncate():
-    model_id="Maykeye/TinyLLama-v0"
-    sequence_length=1024
+
+def _test_prefill_truncate():
+    model_id = "Maykeye/TinyLLama-v0"
+    sequence_length = 1024
 
     model_path = prepare_model(model_id, sequence_length)
     max_new_tokens = 20
 
     generator = AutoGenerator.from_pretrained(
         model_path, revision="", max_batch_size=1, max_sequence_length=sequence_length
     )
-    input_text = "This is something I will tell by the end of the story"
+    input_text = "And to finish the story, I will say that"
 
     request = create_request(id=0, inputs=input_text, max_new_tokens=max_new_tokens, do_sample=False)
     batch = Batch(id=0, requests=[request], size=1, max_tokens=sequence_length)
     generations, _ = generator.prefill(batch)
     assert len(generations) == 1
-    assert generations[0].tokens.ids == [31843]
-    assert generations[0].tokens.texts == ["."]
-
+    assert generations[0].tokens.ids == [357]
+    assert generations[0].tokens.texts == [" it"]
     # Now re-test but with truncate
     generator.clear()
 
     request = create_request(id=0, inputs=input_text, max_new_tokens=max_new_tokens, do_sample=False)
     # This will only leave last tokens
-    request.truncate = 6
+    request.truncate = 3
     batch = Batch(id=0, requests=[request], size=1, max_tokens=sequence_length)
     generations, _ = generator.prefill(batch)
     assert len(generations) == 1
-    assert generations[0].tokens.ids == [291]
-    assert generations[0].tokens.texts == [" and"]
+    assert generations[0].tokens.ids == [266]
+    assert generations[0].tokens.texts == [" the"]
+
+
+def test_prefill_truncate_jetstream():
```

Comment:
The two tests are identical in behaviour, but if jetstream is loaded the other test will fail to run correctly because of incompatibility on the dependencies when using some features of pytorch (i.e.: multiprocessing).
I just have two identical tests, but one is going to be run when jetstream is enabled, the other one will be skipped, and when jetstream is disabled it will be the other way around.